build/versionize-docs.sh

```diff
@@ -49,16 +49,16 @@ HTML_PREVIEW_PREFIX="https://htmlpreview.github.io/\?https://github.com/kubernet
 md_dirs=(docs examples)
 md_files=()
 for dir in "${md_dirs[@]}"; do
-  mdfiles+=($( find "${dir}" -name "*.md" -type f ))
+  md_files+=($( find "${dir}" -name "*.md" -type f ))
 done
-for doc in "${mdfiles[@]}"; do
+for doc in "${md_files[@]}"; do
   $SED -ri \
       -e '/<!-- BEGIN STRIP_FOR_RELEASE -->/,/<!-- END STRIP_FOR_RELEASE -->/d' \
       -e "s|(releases.k8s.io)/[^/]+|\1/${NEW_VERSION}|g" \
       "${doc}"
 
   # Replace /HEAD in html preview links with /NEW_VERSION.
-  $SED -ri -e "s|(${HTML_PREVIEW_PREFIX}/HEAD)|${HTML_PREVIEW_PREFIX}/${NEW_VERSION}|" "${doc}"
+  $SED -ri -e "s|(${HTML_PREVIEW_PREFIX}/HEAD)|${HTML_PREVIEW_PREFIX}/tree/${NEW_VERSION}|g" "${doc}"
```

Comment:
Also this regex should be "s|(${HTML_PREVIEW_PREFIX})/HEAD|\1/tree/${NEW_VERSION}|g" (modulo the path change)